app/containers/Applications/ApplicationReviewContainer.tsx

```diff
@@ -0,0 +1,156 @@
+import React, {useState} from 'react';
+import {Button, Modal, Form, Row, Col} from 'react-bootstrap';
+import {createFragmentContainer, graphql, RelayProp} from 'react-relay';
+import {ApplicationReviewStatus} from 'ApplicationReviewContainer_applicationReview.graphql';
+import createApplicationReviewMutation from '../../mutations/application/createApplicationReviewMutation';
+
+interface Props {
+  relay: RelayProp;
+  applicationReview: any;
```

Comment:
You should be able to find the type of this in `ApplicationReviewContainer_applicationReview.graphql`